test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -317,6 +345,36 @@ private V1beta1CustomResourceDefinition getPravegaCRD() {
 
     }
 
+    private static V1Secret getTLSSecret() {
+        String data = "";
+        String yamlInputPath = "secret.yaml";
+        try (InputStream inputStream = Utils.class.getClassLoader().getResourceAsStream(yamlInputPath)) {
+            data = IOUtils.toString(inputStream, StandardCharsets.UTF_8);
+        } catch (IOException e) {
+            log.error("Could not read from: {}, data:{}", yamlInputPath, data);
```

Comment:
The exception itself is eaten up here. This should be throwing an exception or handle it so that the flow below doesn't execute.